tests/inc/Editor/DataBinding/BlockBindingsTest.php

```diff
@@ -0,0 +1,380 @@
+<?php declare(strict_types = 1);
+
+namespace RemoteDataBlocks\Editor\DataBinding;
+
+use RemoteDataBlocks\Tests\Mocks\MockWordPressFunctions;
+
+/**
+ * Mock the global WordPress functions in implementation namespace.
+ */
+function get_query_var( string $var, $default = '' ): mixed {
+	return MockWordPressFunctions::get_query_var( $var, $default );
+}
+
+// phpcs:disable Universal.Namespaces.OneDeclarationPerFile.MultipleFound
+namespace RemoteDataBlocks\Tests\Editor\DataBinding;
+
+use PHPUnit\Framework\TestCase;
+use Mockery;
+use RemoteDataBlocks\Editor\BlockManagement\ConfigStore;
+use RemoteDataBlocks\Editor\DataBinding\BlockBindings;
+use RemoteDataBlocks\Tests\Mocks\MockQueryRunner;
+use RemoteDataBlocks\Tests\Mocks\MockWordPressFunctions;
+use RemoteDataBlocks\Tests\Mocks\MockQueryContext;
+
+class BlockBindingsTest extends TestCase {
+	private const MOCK_BLOCK_NAME = 'test/block';
+	private const MOCK_OPERATION_NAME = 'test-operation';
+
+	private const MOCK_INPUT_SCHEMA = [
+		'test_input_field' => [
+			'name' => 'Test Input Field',
+			'type' => 'string',
+		],
+	];
+
+	private const MOCK_OUTPUT_SCHEMA = [
+		'is_collection' => false,
+		'mappings' => [
+			'output_field' => [
+				'name' => 'Output Field',
+				'type' => 'string',
+				'path' => '$.output_field',
+			],
+		],
+	];
+	private const MOCK_OUTPUT_FIELD_NAME = 'output_field';
+	private const MOCK_OUTPUT_FIELD_VALUE = 'Test Output Value';
+	private const MOCK_OUTPUT_QUERY_RESULTS = [
+		'is_collection' => false,
+		'results' => [
+			[
+				'result' => [
+					self::MOCK_OUTPUT_FIELD_NAME => [
+						'value' => self::MOCK_OUTPUT_FIELD_VALUE,
+					],
+				],
+			],
+		],
+	];
+
+	protected function tearDown(): void {
+		parent::tearDown();
+		Mockery::close();
+		MockWordPressFunctions::reset();
+	}
+
+	/**
+	 * @runInSeparateProcess
+	 */
```

Comment:
Had to add the [`@runInSeparateProcess`](https://docs.phpunit.de/en/10.5/annotations.html#runinseparateprocess) decorator to prevent static method mocks for the `ConfigStore` class from causing side effects in other tests. Although using dependency injection instead of static methods could be a better approach, it would require significant changes due to the heavy use of static methods in this codebase. Open to suggestions for better ways to handle static methods in tests.